src/components/EmailSubscription/EmailSubscription.scss

```diff
@@ -1,3 +1,40 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/responsiveness";
+@use "~stylesheets/typography";
 
-/* Delete these lines when you begin development */
+.EmailSubscription {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  width: 100%;
+
+  .SearchBox {
+    font-family: "Montserrat Italic";
+    font-size: 36px;
+    line-height: 44px;
+    color: var(--grey);
+    border: none;
+
+    border-radius: 50px;
+    margin-right: var(--extra-extra-large-spacer);
+    padding: var(--small-spacer) var(--extra-large-spacer);
+    width: 765px;
+    min-width: 470px;
+    max-height: 60px;
+
+    @include responsiveness.respond(phone-narrow) {
+      min-width: unset;
+      width: 100%;
+      text-overflow: ellipsis;
+    }
+  }
+
+  .InputButton {
+    background: var(--wedgewood);
+    border-radius: 50%;
+    padding: var(--medium-spacer);
+    border: none;
+  }
+  .FeedbackMessage {
+    @include typography.title-info; // Will change to 'description' after Will's PR
+  }
```

Comment:
Let's use the BEM Naming Conventions for these. I looked at the TS file and it looks like the nested props all live under `EmailSubscription`. Let's rename them to `EmailSubscription__inputButton`, etc. We can keep the nestings here and just make use of the ampersand convention that SCSS provides. `&__inputButton`

Reply:
I'm not sure what you mean by using the ampersand convention, because each subcomponent doesn't inherit the default `EmailSubscription` class, but take a look at the revisions that I made, and let me know if there's anything else that I need to do.